### DISCUSSION TITLE
[ Section 3 - 8 Modules]

### DISCUSSION THREAD

**Discussion by caiojra** [chgasparoto/curso-aws-com-terraform#53] [Q&A]
Olá boa noite, 

estou seguindo o curso mas quando fui criar o Bucket pelo Terraform deu o seguinte erro:

`aws_s3_bucket.my-test-bucket: Creating...

Error: Error creating S3 bucket: AccessDenied: Access Denied
        status code: 403, request id: WW4AS1JF2MR4YXY1, host id: hNcpGP7PU8c/jh6QIpgZOU/05Q2v1LnGkbtt9YMBy2Z5sxmMXxM3jd8dpOtfzSHVhIYh6uSUDBM=

  on main.tf line 6, in resource "aws_s3_bucket" "my-test-bucket":
   6: resource "aws_s3_bucket" "my-test-bucket" {`

Como devo proceder?

**Comment by chgasparoto** [comment]
@caiojra consegue postar seu código? Só pela mensagem de erro significa que o Terraform não tem permissão para criar o recurso utilizando as credenciais que foram passadas.

**Reply by caiojra** [reply]
```hcl
provider "aws" {
    region = "us-east-1" 
    profile = "default"
    }

resource "aws_s3_bucket" "my-test-bucket" {
  bucket = "my-tf-test-bucket-9360897613254568002346432"
  acl    = "private"

  tags = {
    Name        = "My bucket"
    Environment = "Dev"
    Managedby = "Terraform"
  }
}
```
